src/multiversioning.zig

```diff
@@ -233,12 +279,21 @@ pub const MultiversionHeader = extern struct {
             }
 
             for (sizes) |s| if (s == 0) return error.InvalidPastReleases;
+            for (flags) |f| if (f.padding != 0) return error.InvalidPastReleases;
```

Comment:
This does mean that we can't add flags and have the rest be readable by previous version, though...